openstack/orchestration/v1/stacks/requests.go

```diff
@@ -133,20 +139,6 @@ type AdoptOpts struct {
 	Name string
 	// (REQUIRED) The timeout for stack creation in minutes.
 	Timeout int
-	// (OPTIONAL; REQUIRED IF Template IS EMPTY) The URL of the template to instantiate.
```

Comment:
`stack-adopt` has been changed so that it does not accept `templateURL` or `template`. This change was made long ago but the documentation has not been updated. I will file a bug. The heatclient does not accept either `templateURL` or `template`. This commit: https://github.com/openstack/python-heatclient/commit/a34f105ce1096dbdd35568f2289119fefd686701 shows when the option was removed from heatclient